src/lib/structures/profile.mli

```diff
@@ -0,0 +1,28 @@
+(******************************************************************************)
+(*                                                                            *)
+(*     Alt-Ergo: The SMT Solver For Software Verification                     *)
+(*     Copyright (C) 2020-2020 --- OCamlPro SAS                               *)
+(*                                                                            *)
+(*     This file is distributed under the terms of the license indicated      *)
+(*     in the file 'License.OCamlPro'. If 'License.OCamlPro' is not           *)
+(*     present, please contact us to clarify licensing.                       *)
+(*                                                                            *)
+(******************************************************************************)
```

Comment:
Lack of doc: what is this module ? what use does it have ?

Reply:
The doc helps, but why is this module called `Profile` ? Naively, the name would make me think of progiling information, which this is definitely not.